internal/experiments/experiments.go

```diff
@@ -32,8 +41,11 @@ var (
 	EnvPrecedence   Experiment
 )
 
+var ExperimentConfig ExperimentConfigFile
```

Comment:
I don't think this needs to be exported. The resolved values are already available to any user of the package.